src/helpers/makeArc.ts

```diff
@@ -0,0 +1,60 @@
+export interface ArcProps {
```

Comment:
Are you still using makeArc? If not, let's remove it

Reply:
I thought I did but I stashed the chagned yesterday so it was not used. I will remove it now